src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable{
+    public final String color;
```

Comment:
Consider making the 'color' field protected instead of public to adhere to encapsulation principles. Public fields can be accessed directly, which might not be desirable.

Reply:
Ok